src/newViews/ProjectEdit/ProjectDetailsForm/StakeholderList/styles.scss

```
@@ -0,0 +1,39 @@
+.stakeholder-list {
+    .heading {
+        font-size: var(--dui-font-size-heading-extra-small);
+        font-weight: var(--dui-font-weight-medium);
+    }
+
+    .content {
+        display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+
+        .stakeholder-details {
+            display: flex;
+            align-items: center;
+
+            .logo-container {
+                display: flex;
+                align-items: flex-start;
+                flex-shrink: 0;
+                justify-content: center;
+                padding: var(--dui-spacing-extra-small) var(--dui-spacing-small);
+
+                .icon {
+                    width: 1.5rem;
+                    height: 1.5rem;
```

We should use already existing / new variable for `rem`
```suggestion
                    width: 1.5em;
                    height: 1.5em;
```